Tes avi

fahrizalrahman opened the pull request:
Pull Request
- (Cetak penjualan & Surat jalan -> Menggunakan Konversi contoh : 1 dus (60 pcs) )
- Perbaikan modal detail data pembelian (potangan & subtotal kolom tidak sesuai )

fahrizalrahman committed:
review code a afif (diskon bertingkat)

 hapus query tidak perlu

fahrizalrahman committed:
perbaikan download penjualan detail

perbaikan download penjualan detail tidak tampil

haidarafif0809 committed:
Perbaikan Usia Plafon 

- patokan nya di ubah tadi nya dari jatuh tempo, di ubah jadi tanggal transaksi

haidarafif0809 committed:
Perhitungan Diskon Bertingkat Keyup Edit Penjualan

fahrizalrahman committed:
Merge remote-tracking branch 'origin/tes_avi' into tes_avi

fahrizalrahman committed:
cetak penjualan dan cetak surat jalan 

- penambahan satuan konversi contoh : 1 DUS (130 PCS)

fahrizalrahman committed:
perbaikan datatable data pembelian

- salah penempatan potongan & subtotal

RindangRamadhan commented:
REVIEW Code Fahrizal (Rama)

1. cetak_penjualan_tunai_besar.php DAN cetak_penjualan_piutang.php DAN cetak_lap_penjualan_piutang.php DAN cetak_lap_penjualan_tunai_besar.php
	- $potongan = $data0['potongan']; dihapus saja, karena tidak dipakai
	- $select_perusahaan = $db->query("SELECT * FROM perusahaan "); (tolong dibenahi, pilih yang diperlukan saja)

2. cetak_penjualan_surat_jalan.php
	- $select_perusahaan = $db->query("SELECT * FROM perusahaan "); (tolong dibenahi, pilih yang diperlukan saja)

fahrizalrahman committed:
pembenahan review kode rama 

- penghapusan query tidak perlu dan select * diperbaiki (cetak penjualan & surat jalan)
